packages/roosterjs-content-model/lib/modelApi/common/mergeModel.ts

```diff
@@ -42,6 +43,13 @@ export interface MergeModelOption {
      * @default false
      */
     mergeCurrentFormat?: boolean;
+
+    /**
+     * When set to true, segment format of the insert position will be set into the content that is merged into current model.
+     * If the source model already has fontWeight, Italic or underline, it will not be overwritten.
+     * @default false
+     */
+    applyCurrentFormat?: boolean;
```

Comment:
This optionis conflict with `mergeCurrentFormat`, and the two names together make me confused.

Can we merge them into one option? You can make its value to be a string literal type instead of boolean.